pkgs/misc/vim-plugins/generated.nix

```diff
@@ -2157,6 +2157,17 @@ let
     };
   };
 
+  notational-fzf-vim = buildVimPluginFrom2Nix {
+    pname = "notational-fzf-vim";
+    version = "2019-12-03";
+    src = fetchFromGitHub {
```

Comment:
the update script will automatically determine the rev based on the date. Please change commits message to:
```
vimPlugins.notational-fzf-vim: init at 2019-12-03
```

Reply:
Fixed :)